MaterialDesignThemes.Wpf/StylizedBehaviors.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.Xaml.Behaviors;
+
+namespace MaterialDesignThemes.Wpf;
+
+internal class StylizedBehaviors
```

Comment:
I am wondering if we should consider making this `public` and renaming it to `BehaviorsAssist` to match our other classes. If we are going to take a dependency on the behaviors NuGet it might be a useful feature to expose. Thoughts?

Reply:
Could be useful. There is also [this open issue](https://github.com/dotnet/wpf/issues/148) on dotnet/wpf which is requesting to have native support for it.

Reply:
O nice, I had not seen that open issue. I think for now let's go ahead and make it public. I know I have built this same type of class several times on projects and it would be a nice feature to have included.

Reply:
Made it public and added a small code comment hinting at its origin. Please feel free to add additional stuff in there if you see fit.